theano/gpuarray/tests/test_subtensor.py

```diff
@@ -14,11 +14,14 @@
                          GpuAdvancedSubtensor,
                          GpuAdvancedIncSubtensor1,
                          GpuAdvancedIncSubtensor1_dev20,
-                         GpuDiagonal)
+                         GpuDiagonal,
+                         GpuAllocDiag)
 from ..type import gpuarray_shared_constructor
 
 from .config import mode_with_gpu
 
+import pdb
+
```

Comment:
I'll delete these after it passes travis... For now I just keep them there for my local tests.